.github/workflows/sel4test-hw.yml

```
@@ -1,113 +1,26 @@
-# Copyright 2021, Proofcraft Pty Ltd
+# Copyright 2024, Proofcraft Pty Ltd
 #
 # SPDX-License-Identifier: BSD-2-Clause
 
-# sel4test hardware builds and runs
+# hardware builds and runs
 #
-# See sel4test-hw/builds.yml in the repo seL4/ci-actions for configs.
+# See sel4test-hw/builds.yml and sel4bench-hw/builds.yml in the repo
+# seL4/ci-actions for configs.
 
-name: seL4Test-HW
+name: HW
 
 on:
   # needs PR target for secrets access; guard by requiring label
   pull_request_target:
-    types: [opened, reopened, synchronize, labeled]
-
-# downgrade permissions to read-only as you would have in a standard PR action
-permissions:
-  contents: read
-
-# To reduce the load (especiually on the machine queue) we cancel any older runs
-# of this workflow for the current PR. Such runs exist, if there were new pushes
-# to the PR's branch without waiting for the workflow to finish. As a side
-# effect, pushing new commits now becomes a convenient way to cancel all the
-# older runs, e.g. if they are stuck and would only be stopped by the timeout
-# eventually.
-# Note that we could do the concurrency handling at a finer level, and only wrap
-# the actual run on the hardware. But there seems not much gain in letting the
-# older builds run, as these are usually obsolete with new pushes also.
-concurrency:
-  group: ${{ github.workflow }}-pr-${{ github.event.number }}
-  cancel-in-progress: true
+    types: [synchronize, labeled]
```

Was it intentional no drop `opened` and `reopened` here?
```suggestion
    types: [opened, reopened, synchronize, labeled]
```

---

Yes, the idea was that for `opened`, the test can't trigger because there is no label yet, and for `reopened` that it first should be updated (merge from master or otherwise triggering `synchronize`) before it makes sense to run hardware tests again.

I might be wrong about the trigger at `opened`, because it's possible that there will be only one event if the label is added while opening the PR. If that is the case and is annoying, I'd be happy to add that one back in.